python/lsst/ts/criopy/gui/sal/topic_window.py

```diff
@@ -162,7 +167,17 @@ def current_field_changed(self, field_index: int) -> None:
             self._set_unknown()
             return
         self._change_field(topic_index, field_index)
-        self.collection.topics[topic_index].selectedField = field_index
+        self.collection.topics[topic_index].selected_field = field_index
+
+    def field_changed(self, field: TopicField) -> None:
+        """Should be overwritten in child classes to setup scaling etc.
+
+        Parameters
+        ----------
+        field : TopicField
+            New topic.
+        """
+        pass
```

Comment:
You can raise `NotImplementedError`.

Reply:
Some of the descendants don't override the field_changed method, so raising the exception will not fit. I changed comment line to "May be overwitten.."